install.sh

```diff
@@ -656,7 +656,20 @@ do_install() {
 					pkgs="$pkgs docker-compose-plugin docker-ce-rootless-extras$pkg_version"
 				fi
 				if version_gte "23.0"; then
-						pkgs="$pkgs docker-buildx-plugin docker-model-plugin"
+						case "$lsb_dist.$dist_version" in
```

Comment:
Also, we should avoid having to maintain this list of distros. We definitely don't want to end up having adding each new distro to the list.
We should default to installing all plugins (after a version check), and then only disable for the distros we know that are not supported.